src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.props

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <!--
+    Settings users may update as they see fit.
+  -->
+  <PropertyGroup>
+    <!--
+      Options added to the Open API retrieval tool ('dotnet-getdocument') command line. Available options control
+      console output: 'no-color', 'prefix-output' and 'verbose'. All require a double-dash prefix.
+    -->
+    <OpenApiGetDocumentOptions Condition=" '$(OpenApiGetDocumentOptions)' == '' "></OpenApiGetDocumentOptions>
+
+    <!--
+      If 'true' (the default), enable retrieval of Open API documents. Otherwise, this feature is completely disabled.
+    -->
+    <OpenApiRetrieveDocuments Condition=" '$(OpenApiRetrieveDocuments)' == '' ">true</OpenApiRetrieveDocuments>
```

Comment:
Why would I install this package and then disable it's functionality completely? Seems like it would be simpler to say "if you want to disable the feature, uninstall the package"

Reply:
Users are not usually aware of this package at all. We'll be working with NSwag and SwashBuckle to ship with a dependency on it in their existing NSwag.AspNetCore and SwashBuckle.AspNetCore. Those packages have a lot of features and some users may wish to disable this particular one.

That is, we decided on opt-out for saving Open API docs after build.